### PR TITLE
DBE copy changes

### PR DESCRIPTION
**Pull request by rudigiesler**
Copy changes to ensure copy is the same as in the sheet https://docs.google.com/spreadsheets/d/1cVjPb-ae1qOh4wxJ2skm0kS4QrUoCp9opm9Pqd59VGc/edit#gid=861878833

### CODE REVIEW

File: dbe/domain-eng.yml [hunk 1]
```diff
@@ -133,10 +133,10 @@ responses:
       - text: |
           Reply with one of the words in BOLD to continue
 
-          *ACCEPT* ✅ - _to accept the terms and conditions_
-          *MORE* 📄 - _to read the terms and conditions_
+          ACCEPT ✅ - _to accept the terms and conditions_
+          MORE 📄 - _to read the terms and conditions_
 
-          (or Reply *MENU* 📌 _to return to the main menu_)
+          (or Reply MENU 📌 _to return to the main menu_)"
```
Comment (erikh360) [inline dbe/domain-eng.yml:139]: The double quote at the end looks out of place.

File: dbe/domain-eng.yml [hunk 2]
```diff
@@ -133,10 +133,10 @@ responses:
       - text: |
           Reply with one of the words in BOLD to continue
```
Comment (erikh360) [inline dbe/domain-eng.yml:134]: Won't this change mean it's not bold anymore?

**Review by erikh360 — COMMENTED** [review]
Looks good just the two comments

File: dbe/domain-eng.yml (hunk 1 above)
Comment (rudigiesler) [reply dbe/domain-eng.yml:139]: Yes thanks. When I copy from the spreadsheet to my editor it adds `"`, I missed this one.

File: dbe/domain-eng.yml (hunk 2 above)
Comment (rudigiesler) [reply dbe/domain-eng.yml:134]: Yup, I asked about it but was told to stick with what's in the sheet.

**Pull request merged**